hq/app/schedule/IamRemovePassword.scala

```diff
@@ -0,0 +1,32 @@
+package schedule
+
+import aws.AwsAsyncHandler.awsToScala
+import aws.iam.IAMClient.SOLE_REGION
+import aws.{AwsClient, AwsClients}
+import com.amazonaws.services.identitymanagement.AmazonIdentityManagementAsync
+import com.amazonaws.services.identitymanagement.model.DeleteLoginProfileRequest
+import model.{AwsAccount, IamAuditUser}
+import play.api.Logging
+
+import scala.concurrent.ExecutionContext
+import scala.util.{Failure, Success}
+
+object IamRemovePassword extends Logging {
+
+  def deleteUserLoginProfile(client: AwsClient[AmazonIdentityManagementAsync], user: IamAuditUser)(implicit ec: ExecutionContext): Unit = {
+    val request = new DeleteLoginProfileRequest().withUserName(user.username)
+    awsToScala(client)(_.deleteLoginProfileAsync)(request).onComplete {
+      case Failure(exception) => logger.warn(s"failed to delete password for user ${user.username} in ${user.awsAccount} account.", exception)
+      case Success(result) => logger.info(s"successfully deleted password for user ${user.username} in ${user.awsAccount} account. DeleteLoginProfile Response: ${result.toString}.")
+    }
```

Comment:
We could add markers to these log lines to enable us to create visualisations in ELK - number of items deleted per account over time, for example.

Reply:
Nice idea 👍🏽. 

Hopefully this shouldn't happen too often so this could be something we consider for a different SHQ feature. Thanks for the tip.